llabake/Hello-Books
# Ft favorite books page react redux 154496475

What does this PR do?
It creates the user's favorite book page

Description of Task to be completed?
User should be able to navigate the their favorite books page, view it and delete a book from the list

How should this be manually tested?
clone the repo
cd into the project directory
Run npm install to install all the dependency
Run npm run dev
visit the port where the webpack-dev-server started
on successful login, on navigation to the profile page, the user profile page should be loaded there
What are the relevant pivotal tracker stories?
154496475

Any background context you want to add?
N/A
Screenshots
![image](https://user-images.githubusercontent.com/29184825/35290779-948ea6fe-006b-11e8-927b-5e9d816711ec.png)
